components/raftstore-v2/src/router/message.rs

```diff
@@ -195,8 +202,14 @@ impl fmt::Debug for StoreMsg {
     fn fmt(&self, fmt: &mut fmt::Formatter<'_>) -> fmt::Result {
         match *self {
             StoreMsg::RaftMessage(_) => write!(fmt, "Raft Message"),
+            StoreMsg::PeerCreation(_) => write!(fmt, "Peer Creation"),
             StoreMsg::Tick(tick) => write!(fmt, "StoreTick {:?}", tick),
             StoreMsg::Start => write!(fmt, "Start store"),
         }
     }
 }
+
+pub struct PeerCreation {
```

Comment:
Should be defined in admin module. And there should be one `Box` instead of two.

Reply:
Good suggestion.